static/js/chart/draw.ts

```diff
@@ -616,16 +509,18 @@ function drawLineChart(
         .data(dataset)
         .enter()
         .append("circle")
-          .attr("class", "dot")
-          .attr("cx", (d, i) => xScale(d[0]))
-          .attr("cy", d => yScale(d[1]))
-          .attr("fill", COLORS[i])
-          .attr("r", 3);
+        .attr("class", "dot")
+        .attr("cx", (d, i) => xScale(d[0]))
+        .attr("cy", (d) => yScale(d[1]))
+        .attr("fill", COLORS[i])
```

Comment:
the color here should match the color in the legend too (pull up the getColorfn). i'll update that later to make it work for the line charts too